x/mongo/driver/topology/topology_options_test.go

```diff
@@ -76,3 +77,34 @@ func TestDirectConnectionFromConnString(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadBalancedFromConnString(t *testing.T) {
+	testCases := []struct {
+		name         string
+		uriOptions   string
+		loadBalanced bool
+	}{
+		{"loadBalanced=true", "loadBalanced=true", true},
+		{"loadBalanced=false", "loadBalanced=false", false},
+		{"loadBalanced", "", false},
```

Comment:
[nit] maybe the name should be `loadBalanced unset`?

Reply:
Done.